tensorflow_addons/__init__.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Useful extra functionality for TensorFlow maintained by SIG-addons"""
+
```

Comment:
We'd better `import future` first, and then delete them at the end, like what to do in https://github.com/tensorflow/tensorflow/blob/6b17f1649ead3bdcda3639dbdd9e21e50d8856fb/tensorflow/__init__.py#L21